packages/settingregistry/src/tokens.ts

```diff
@@ -591,4 +603,63 @@ export namespace ISettingRegistry {
      */
     selector: string;
   }
+
+  /**
+   * An interface describing a toolbar item.
+   */
+  export interface IToolbarItem extends PartialJSONObject {
+    /**
+     * Unique toolbar item name
+     */
+    name: string;
+
+    /**
+     * The command to execute when the item is triggered.
+     *
+     * The default value is an empty string.
+     */
+    command?: string;
+
+    /**
+     * The arguments for the command.
+     *
+     * The default value is an empty object.
+     */
+    args?: PartialJSONObject;
+
+    /**
+     * Whether the toolbar item is disabled. `false` by default.
+     *
+     * #### Notes
+     * This allows an user to suppress toolbar items.
+     */
+    disabled?: boolean;
```

Comment:
If I understand it correctly this decides whether the button is shown or not. The name `disabled` might clash with the `disabled` state of the button or with the `disabled` state of the command which have different semantic meaning (violating the principle of the least surprise for extension developers). It might be better to rename it to `hidden`. It is not a strong opinion.

Reply:
...but I also see an argument for keeping `disabled` for consistency with menus. So I am really not sure which way is better.

Reply:
Indeed I kept the word `disabled` for consistency with the shortcuts and menus.
`hidden` could be misleading too as the item node is even not in the dom and it is somehow also conflicting with the `isVisible` attribute of a command.

So although I don't have a strong opinion neither, I would rather prefer to keep `disabled`. But I will update the doc string to mention something like:

```
Whether the toolbar item is ignored (i.e. not created) or not.
```